presidio-anonymizer/presidio_anonymizer/operators/custom.py

```diff
@@ -36,3 +35,10 @@ def operator_name(self) -> str:
     def operator_type(self) -> OperatorType:
         """Return operator type."""
         return OperatorType.Anonymize
+
+    def _call_lambda(self, text: str = None, params: Dict = None):
+        new_val = params.get(self.LAMBDA)
+        sig = signature(new_val)
+        if len(sig.parameters) == 1:
+            return new_val(text)
+        return new_val(text, params.get("entity_type"))

```

Comment:
Are there cases where we'd want more than just the entity type? It could be worth thinking about other use cases for the custom lambda where the user might need additional metadata.

Reply:
How about start location and score? I think it could be a dictionary in case we want to add more stuff in the future.

Reply:
Let's discuss offline. I was thinking that maybe a better solution would be to create a new operator. Wdyt?

Reply:
Agree! that'll be cleaner :D